pubtools/_quay/iib_operations.py

```diff
@@ -169,8 +170,18 @@ def _sign_index_image(
     current_signatures: list[tuple[str, str, str]] = [
         (e.reference, e.digest, e.signing_key) for e in to_sign_entries
     ]
+    if pre_push:
```

Comment:
Should this be here?

Reply:
you mean if it these methods shouldn't be available on the global level?

Reply:
I'm trying to understand this code. My best guess: if pre_push is true, only use prepush signers, if it's false only  use pust push signers. If that's so, why not just:
SIGNER_BY_LABEL[signer["label"]].pre_push == pre_push

Reply:
yeah, it can be simplified like that